testsuite/MDAnalysisTests/test_api.py

```diff
@@ -49,3 +52,33 @@ def test_ResidueGroup():
 
 def test_SegmentGroup():
     assert mda.SegmentGroup is mda.core.groups.SegmentGroup
+
+
+@pytest.mark.parametrize('module',
+                         ('',
+                          '.analysis',
+                          '.analysis.data.filenames',
+                          '.analysis.distances',
+                          '.analysis.encore',
+                          '.analysis.encore.dimensionality_reduction',
+                          '.analysis.hbonds',
+                          '.analysis.hydrogenbonds',
+                          '.coordinates',
+                          '.core',
+                          '.lib._augment',
+                          '.lib._cutil',
+                          '.lib.formats',
+                          '.lib.pkdtree',
+                          '.topology',
+                          '.topology.tpr',
+                          '.visualization'
+                          )
+                         )
+def test_import_star(module):
+    try:
+        # This is really bad and ugly but I haven't found a way to run this
+        # using `importlib.import_module...`
+        exec("from MDAnalysis{} import *".format(module))
```

Comment:
As written in the comment above this is bad practice but after testing a lot I haven't found a way to run something like `from module import *` using the `importlib` package... I'm open for ideas

Reply:
Another approach is the one we use for datafiles: https://github.com/MDAnalysis/mdanalysis/blob/76d9ca1bd0b48cf692b6b3d720fa3a85dc13ef98/testsuite/MDAnalysisTests/utils/test_datafiles.py#L34 it checks that all the names in `__all__` are also in `dir`. This would avoid the `exec` that may break the coverage report.